depinject/README.md

```diff
@@ -7,15 +7,16 @@ to simplify the definition of a blockchain by replacing most of app.go's boilerp
 
 ## Usage
 
-### `depinject` example 
+`depinject` includes an expressive and composable [Configuration API](https://pkg.go.dev/github.com/cosmos/cosmos-sdk/depinject#Config).
+A core configuration is `Provide`, for example this code snippet
 
 ```go
 package main
 
 import (
 	"fmt"
 
-	"cosmossdk.io/depinject"
+	"github.com/cosmos/cosmos-sdk/depinject"
```

Comment:
why was this reverted?

Reply:
intellij's smart parse of golang code blocks in markdown fails on `cosmossdk.io`, I thought it was a mistake.  If not I can revert my revert.